### ISSUE TITLE
BatchQuery ignores property mapping

### ISSUE THREAD

**Issue by Christoph-Wagner** [mikependon/RepoDB#10]
Using `Query` works fine, but for `BatchQuery` the `MapAttribute` is ignored.

Here is a simple example: 

```SQL
CREATE TABLE [dbo].[Test](
	[Id] [int] NOT NULL,
	[WrongName] [nvarchar](50) NOT NULL
) ON [PRIMARY]
GO
```

```C#
public class Test : DataEntity {
	public int Id {get;set;}
	[Map("WrongName")]
	public string RightName {get;set;}
}

using (var con = new SqlConnection(ConnectionString).EnsureOpen())
{
  con.Query<Test>(); // works fine
  
  // SqlException: Invalid column name 'RightName'.
  con.BatchQuery<Test>(0,2,OrderField.Parse(new {Id = Order.Descending })); 
}
```

The generated SQL is:

```SQL
WITH CTE
AS (  
    SELECT ROW_NUMBER() OVER (
    ORDER BY [Id] DESC
    ) AS [RowNumber],
    [Id],
    [WrongName] -- correct
  FROM [Test]
  )
SELECT [Id], [RightName] -- not correct
FROM CTE
WHERE ( [RowNumber] BETWEEN 1 AND 2 )
ORDER BY [Id] DESC
```

The Problem is in [SqlDbStatementBuilder.cs, line 61](https://github.com/mikependon/RepoDb/blob/master/RepoDb/RepoDb/SqlDbStatementBuilder.cs#L61) which has `.FieldsFrom(fields)`, the problem stops occuring when the line is replaced with `.FieldsFrom(Command.BatchQuery)`.

I'd send a PR, but with the lag of tests and my lack of deeper understanding I neither know if this will introduce further bugs nor if it has performance penalties.

**Comment by mikependon** [comment]
Hi Christoph, I thanked you for this. This is the first bug reported on the library outside of my personal testing. I will create a test case for this scenario, and this will be part of the next beta release. Will update this bug from time-to-time.

**Comment by mikependon** [comment]
@Christoph-Wagner - my unit test test caught this one as well. Issue has been fixed at the master code.

Fixed At:
At Line 61:
https://github.com/mikependon/RepoDb/blob/master/RepoDb/RepoDb/SqlDbStatementBuilder.cs

Unit Test: https://github.com/mikependon/RepoDb/blob/master/RepoDb/RepoDb.Tests/RepoDb.UnitTests/SqlDbStatementBuilderTest.cs

The RepoDb.Core project has also been sync(ed).

Will be a part of the GA release.

**Comment by mikependon** [comment]
Will close this issue now.

**Comment by mikependon** [comment]
The fixed has been a part of our GA release v1.2.0. Thanks
